python/lsst/ts/dimm/utils/conversion.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of ts_dimm.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import math
+
+
+def convert_to_int(value):
+    """Convert a value to an int, or 0 in case the conversion fails.
+
+    Parameters
+    ----------
+    value: `str` or `int`
+        The value to convert.
+
+    Returns
+    -------
+    `int`
+        The value converted to an int, or 0 in case the conversion fails.
+
+    """
+    try:
+        return int(value)
+    except ValueError:
+        return 0
```

Comment:
should be `math.nan`

Reply:
We discussed this on slack and agreed to leave it to `0` since `math.nan` is a float and there is no preferred sentinel value for integers.